File: python/tamrin5.py
#عددی رو دریافت کرده و اعداد زوج کوچکتر از آن را چاپ کند
number=input("enter the numbr:")
number=int(number)
n=2         #شروع از اولین عدد  
while n<number:
    isPrime=True
    i=2
    while i*i<=n:    #برسی مقسوم علیه تا جزر
        if n%i==0:   
            isPrime=False
            break  
               
                  
        i+=1   
    if isPrime:    
        print(n)     
    n+=1
         
         

File: cpp/if/tamrin5.cpp
#include<iostream>// عددی رو از ورودی دریافت کرده و بیان کند چه روزی از هفته  هست( در صورتی که عددی نامعتبر وارد شود پیغام مناسب نمایش داده شود)
using namespace std;

int main()
{
    int num;
    cout<<"enter the number:";
    cin>>num;
    switch (num)
    {
    case 1:
       cout<<"Saturday";
        break;
            case 2:
       cout<<"Sunday";
        break;
            case 3:
       cout<<"Monday";
        break;
            case 4:
       cout<<"Tuesday";
        break;
            case 5:
       cout<<"Wednesday";
        break;
            case 6:
       cout<<"Thursday";
        break;
            case 7:
       cout<<"Friday";
        break;
    
    default:
    cout<<"The entered number is invalid.";
        break;
    }
   
    return 0;
}
